### PR TITLE
Fix logic for whether we're currently catching up

### PR DESCRIPTION
**Pull request by jessepinho**
It's currently backwards.

### CODE REVIEW

File: packages/router/src/grpc/view-protocol-server/status.test.ts [hunk 1]
```diff
@@ -28,7 +28,9 @@ describe('Status request handler', () => {
     };
 
     mockServices = {
-      getWalletServices: vi.fn(() => Promise.resolve({ indexedDb: mockIndexedDb })),
+      getWalletServices: vi.fn(() =>
+        Promise.resolve({ indexedDb: mockIndexedDb }),
+      ) as MockServices['getWalletServices'],
```
Comment (jessepinho) [inline packages/router/src/grpc/view-protocol-server/status.test.ts:33]: Fixes a TypeScript complaint

File: packages/router/src/grpc/view-protocol-server/status.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const status: Impl['status'] = async (_, ctx) => {
   if (!fullSyncHeight) throw new Error('Last block synced not in db');
 
   return {
-    catchingUp: fullSyncHeight === latestBlockHeight,
+    catchingUp: fullSyncHeight !== latestBlockHeight,
```
Comment (grod220) [inline packages/router/src/grpc/view-protocol-server/status.ts:12]: Ah good catch. It's an edge case (maybe an error somewhere else if so), but should we do:

```
catchingUp: fullSyncHeight < latestBlockHeight,
```

**Review by grod220 — APPROVED** [review]

File: packages/router/src/grpc/view-protocol-server/status.ts (hunk 2 above)
Comment (Valentine1898) [reply packages/router/src/grpc/view-protocol-server/status.ts:12]: https://github.com/penumbra-zone/penumbra/blob/d3bbe4c920dd03981ed883d347ed579dae6af363/crates/view/src/service.rs#L326-L334
rust view service also returns `catchingUp=false` if the difference is only 1 block

**Review by Valentine1898 — APPROVED** [review]

File: packages/router/src/grpc/view-protocol-server/status.ts (hunk 2 above)
Comment (grod220) [reply packages/router/src/grpc/view-protocol-server/status.ts:12]: ah, nice find

**Pull request merged**